extensions/podcasts-now/src/add-podcast.tsx

```diff
@@ -0,0 +1,86 @@
+import { Action, ActionPanel, Form, LocalStorage, popToRoot, showToast, Toast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { uniq } from "lodash";
+import { XMLParser } from "fast-xml-parser";
+import fs from "fs";
+
+import { PODCASTS_FEEDS_KEY } from "./constants";
+
+export default function Command() {
+  const [podcastsFeeds, setPodcastsFeeds] = useState<string>("");
+  const [files, setFiles] = useState<string[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const currentFeeds = await LocalStorage.getItem(PODCASTS_FEEDS_KEY);
+      if (typeof currentFeeds === "string") {
+        setPodcastsFeeds(JSON.parse(currentFeeds).join("\n"));
+      } else {
+        setPodcastsFeeds("");
+      }
+    })();
+  }, []);
+
+  const onFilePick = async (values: string[]) => {
+    const filePath = values[0];
+    fs.readFile(filePath, (err, data) => {
+      if (err) {
+        showToast({
+          style: Toast.Style.Failure,
+          title: "Read file error",
+          message: err.message,
+        });
+        return;
+      }
+      const parser = new XMLParser({ ignoreAttributes: false });
+      const parsedFeeds = parser.parse(data.toString("utf8"));
+      const feedUrls = parsedFeeds.opml.body.outline.outline.map((el: { "@_xmlUrl": string }) => el["@_xmlUrl"]);
+      setPodcastsFeeds((prev) => [...(prev || "").split("\n"), ...feedUrls].join("\n"));
+    });
+    setFiles([]);
+  };
+
+  const onSubmit = async ({ urls }: { urls: string }) => {
+    await LocalStorage.removeItem(PODCASTS_FEEDS_KEY);
+    const validUrls = urls.split("\n").filter((url: string) => {
+      try {
+        new URL(url);
+        return true;
+      } catch (_) {
+        return false;
+      }
+    });
+    await LocalStorage.setItem(PODCASTS_FEEDS_KEY, JSON.stringify(uniq(validUrls)));
+    popToRoot({ clearSearchBar: false });
+  };
+
+  return (
+    <Form
+      actions={
+        <ActionPanel title="#1 in raycast/extensions">
```

Comment:
I'm not sure I get what does the title `"#1 in raycast/extensions"` mean here?

Reply:
This is the default title from one of our examples 😊

Reply:
But that doesn't have anything to do with this extension, right? 😊

Reply:
No, just a "leftover" 😄